src/commands/new.rs

```diff
@@ -34,9 +35,14 @@ pub struct TemplateNewCommandCore {
 
     /// The directory in which to create the new application or component.
     /// The default is the name argument.
-    #[clap(short = 'o', long = "output")]
+    #[clap(short = 'o', long = "output", group = "location")]
     pub output_path: Option<PathBuf>,
 
+    /// The directory in which to create the new application or component.
```

Comment:
```suggestion
    /// Create the application in the current working directory.
```

... Or something (comment appears to be the same from `-o`)

Reply:
oopsie boopsie!  Thanks for the catch